counting_sort/java/countingSort.java

```diff
@@ -0,0 +1,28 @@
+import java.io.;
+import java.util.;
+
+
+ public class Solution {
+
+    public static void main(String[] args) {
+         Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp0a3pa2ax/counting_sort/java/countingSort.java
+++ b/tmp/tmp0a3pa2ax/counting_sort/java/countingSort.java
@@ -5,7 +5,7 @@
  public class Solution {
 
     public static void main(String[] args) {
-         Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. 
+         Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution.
         Scanner s=new Scanner(System.in);
         int n=s.nextInt();
         int[] ar=new int[n];
```